.github/CODEOWNERS

```diff
@@ -1,13 +1,17 @@
+# Link to the documentation:
 # https://help.github.com/articles/about-codeowners/
-# please keep this file in the same order as files
 
+# Please keep the entries below in an alphabetical order.
+
+# assets/launcher/lang/sklmessages_cs_CZ.**
 assets/launcher/lang/sklmessages_en_US.** @skmedix
 assets/launcher/lang/sklmessages_eo.** @Waterrail
 assets/launcher/lang/sklmessages_es_ES.** @atesin
 # assets/launcher/lang/sklmessages_fr_FR.**
-assets/launcher/lang/sklmessages_hu_HU.** @Gbr22 
+assets/launcher/lang/sklmessages_hu_HU.** @Gbr22
 assets/launcher/lang/sklmessages_nb_NO.** @SirLexPLAY
 assets/launcher/lang/sklmessages_pl_PL.** @Waterrail @Greg-21 @Indexer
```

Comment:
I would shrink PL CODEOWNERS to one, as GH only supports one owner per folder/file.

Reply:
Yeah, only the first one is getting review request. Probably a bug, but existing since

Reply:
I don't think so

Reply:
So I'll remove also them.

Reply:
So I'm going to remove also them.